### PR TITLE
Update pytorch_rnn_gru_lstm.py

### PR DESCRIPTION
**Pull request by adamroberge**
print(f"Accuracy on training set: {check_accuracy(train_loader, model)*100:2f}") This line had a missing "." for the number of decimals.

### CODE REVIEW

**Comment by Icemagic33** [comment]
Fix typo
